Formula/p/plakar.rb

```diff
@@ -0,0 +1,21 @@
+class Plakar < Formula
+  desc "Effortless Open-Source Backup"
+  homepage "https://plakar.io/"
+  url "https://github.com/PlakarKorp/plakar.git", branch: "main", revision: "main"
+  version "1.0.0"
```

Comment:
This is not a stable version and therefore not eligible for inclusion

Reply:
Since I can't find a mention anywhere about when that would be, I think closing it until such a time makes more sense.